src/Microsoft.Restier.Core/ApiConfiguration.cs

```diff
@@ -34,7 +34,7 @@ namespace Microsoft.Restier.Core
     /// </remarks>
     public class ApiConfiguration
     {
-        private static ConcurrentDictionary<Type, Action<IServiceCollection>> configurations =
+        private static ConcurrentDictionary<Type, Action<IServiceCollection>> _internalServicesCallback =
```

Comment:
I still think we should keep consistence, if you want to keep it please update other places